cmd/apply/cmdapply.go

```diff
@@ -43,14 +43,14 @@ func NewCmdApply(f util.Factory, ioStreams genericclioptions.IOStreams) *cobra.C
 	}
 
 	cmdutil.CheckErr(applier.SetFlags(cmd))
-
 	cmdutil.AddValidateFlags(cmd)
+	_ = cmd.Flags().MarkHidden("validate")
```

Comment:
Do we know that it is safe to ignore the errors here?

Reply:
Do we want to crash? Maybe we could log a warning (once we implement logging). For now, I think this is reasonable.